features/step_definitions/brief_response_steps.rb

```diff
@@ -23,18 +23,30 @@
   page.visit("#{dm_frontend_domain}#{url}")
 end
 
-Given 'that supplier is on that framework' do
-  submit_supplier_declaration(@framework['slug'], @supplier["id"], {})
+Given 'that supplier has applied to be on that framework' do
+  submit_supplier_declaration(@framework['slug'], @supplier["id"], {'status': 'complete'})
+end
+
+Given 'we accept that suppliers application to the framework' do
```

Comment:
These two should probably be `And`s rather than `Given`s, but we don't really have a system for this in the step definitions so let's not worry about it right now.